includes/Checker/Checks/Plugin_Readme_Check.php

```diff
@@ -198,12 +198,67 @@ private function check_stable_tag( Check_Result $result, string $readme_file, Pa
 	private function check_for_warnings( Check_Result $result, string $readme_file, Parser $parser ) {
 		$warnings = $parser->warnings ? $parser->warnings : array();
 
+		// This should be ERROR rather than WARNING. So ignoring here to handle separately.
```

Comment:
```suggestion
		// TODO: This should be ERROR rather than WARNING. So ignoring here to handle separately.
```

Mark it as "TODO" so that others are aware that this should be implemented sooner rather than later.

Reply:
I have already made PR for this. https://github.com/WordPress/plugin-check/pull/398